pkg/cmd/root/pre_command.go

```diff
@@ -260,3 +267,23 @@ func linuxUpdateMessage(f *cmdutil.Factory) error {
 
 	return nil
 }
+
+// 0 = authorization was not asked yet, 1 = accepted, 2 = denied
```

Comment:
just to follow golang's doc convention
```suggestion
// checkAuthorizeMetricsCollection 0 = authorization was not asked yet, 1 = accepted, 2 = denied
```